src/common/syncjournaldb.h

```diff
@@ -106,7 +106,7 @@ class OCSYNC_EXPORT SyncJournalDb : public QObject
             , _valid(false)
         {
         }
-        int _chunk;
+        quint64 _chunk;
```

Comment:
Done.

Reply:
Luckily sqlite doesn't care whether it says INTEGER or INTEGER(8), but the `int64Value` change is indeed critical. It wouldn't break existing installs because `intValue` vs `int64Value` is just about the size of the integer the sqlite value gets stored in.

However, when this behavior changes what will happen to existing half-done up/downloads that used the previous semantics? It would make sense to clear the up/download tables on version jumps.

Reply:
There is actually no reason to change that.  The chunk is only used in the database for the old chunking algo.   So please leave it as an int here.